app/Models/Missions/Mission.php

```diff
@@ -904,6 +904,20 @@ public function briefingFactions()
         return $filledFactions;
     }
 
+    private function parseFactionId($rawFaction) 
+    {
+        //Briefings are per faction numbers
+        if(is_int($rawFaction)) return $rawFaction;
+
+        //Briefings use a different system so just dirtily attempt to parse them
+        if(str_contains($rawFaction, 'blu')) return 0;
+        if(str_contains($rawFaction, 'opf')) return 1;
+        if(str_contains($rawFaction, 'ind')) return 2;
+        if(str_contains($rawFaction, 'civ')) return 3;
```

Comment:
This won't be able to check non-vanilla factions, if people use custom orbats and similar.

Reply:
I feel this brings us closer to what we used to use. I'm not sure which system is a better one. If someone inputs a non defined faction what will it return? Is it the same as what someone entered or will it error out? @BorderKeeper

Reply:
With unknown factions defaulting to civilian i feel this is a good temporary measure.